crates/solidity/inputs/schema/grammar/topics/05-expressions/01-base-expressions.yml

```diff
@@ -2,28 +2,51 @@
 
 - name: "Expression"
   kind: "Rule"
-  unversioned:
-    parserType: "Precedence"
-    choice:
-      - reference: "AssignmentExpression"
-      - reference: "ConditionalExpression"
-      - reference: "OrExpression"
-      - reference: "AndExpression"
-      - reference: "EqualityComparisonExpression"
-      - reference: "OrderComparisonExpression"
-      - reference: "BitOrExpression"
-      - reference: "BitXOrExpression"
-      - reference: "BitAndExpression"
-      - reference: "ShiftExpression"
-      - reference: "AddSubExpression"
-      - reference: "MulDivModExpression"
-      - reference: "ExponentiationExpression"
-      - reference: "UnarySuffixExpression"
-      - reference: "UnaryPrefixExpression"
-      - reference: "FunctionCallExpression"
-      - reference: "MemberAccessExpression"
-      - reference: "IndexAccessExpression"
-      - reference: "PrimaryExpression"
+  versioned:
+    0.4.11:
+      parserType: "Precedence"
+      choice:
+        - reference: "AssignmentExpression"
+        - reference: "ConditionalExpression"
+        - reference: "OrExpression"
+        - reference: "AndExpression"
+        - reference: "EqualityComparisonExpression"
+        - reference: "OrderComparisonExpression"
+        - reference: "BitOrExpression"
+        - reference: "BitXOrExpression"
+        - reference: "BitAndExpression"
+        - reference: "ShiftExpression"
+        - reference: "AddSubExpression"
+        - reference: "MulDivModExpression"
+        - reference: "ExponentiationExpression"
+        - reference: "UnarySuffixExpression"
+        - reference: "UnaryPrefixExpression"
+        - reference: "FunctionCallExpression"
+        - reference: "MemberAccessExpression"
+        - reference: "IndexAccessExpression"
+        - reference: "PrimaryExpression"
+    0.6.0:
```

Comment:
not sure why we are duplicating versions for the same contents?

Reply:
a hack to get the versions of the exponentiation expressions to be used. When the schema is changed this will be resolved

Reply:
Sure, we can put it aside for now. In that case, I suggest adding a `TODO` here to explain for the future.